pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -647,6 +647,7 @@ func PublishMetricsToZedCloud(ctx *zedagentContext, cpuMemoryStat [][]string,
 			ReportAppMetric.Network = append(ReportAppMetric.Network,
 				networkDetails)
 		}
+		log.Infof("XXX ReportMetrics: network %+v", ReportAppMetric.Network)
```

Comment:
Same here.. Will we keep spewing this everytime we send metrics? Not very useful.. right? Retain it as debug?

Reply:
Removed